### ISSUE TITLE
Rising error if stdout gets closed (e. g. pipe to head)

### ISSUE THREAD

**Issue by 1-0** [fail2ban/fail2ban#2758]
```
root@f:/etc/fail2ban# fail2ban-client status sshd|head -n 8
Status for the jail: sshd
|- Filter
|  |- Currently failed: 0
|  |- Total failed:     69
|  `- File list:        /var/log/auth.log
`- Actions
   |- Currently banned: 2557
   |- Total banned:     2557
2020-06-19 08:39:51,711 fail2ban                [117232]: ERROR   [Errno 32] Broken pipe
Traceback (most recent call last):
  File "/usr/bin/fail2ban-client", line 37, in <module>
    exec_command_line(sys.argv)
  File "/usr/lib/python3/dist-packages/fail2ban/client/fail2banclient.py", line 502, in exec_command_line
    exit(255)
  File "/usr/lib/python3/dist-packages/fail2ban/client/fail2bancmdline.py", line 317, in exit
    sys.stdout.flush()
BrokenPipeError: [Errno 32] Broken pipe
Exception ignored in: <_io.TextIOWrapper name='<stdout>' mode='w' encoding='utf-8'>
BrokenPipeError: [Errno 32] Broken pipe
```

**Comment by sebres** [comment]
Fixed in 39d4bb3c35ffb3bc6cdead5ecb58b3377f87867c

**Comment by sebres** [comment]
The issue with broken pipe error by output from fail2ban-regex fixed ultimately in c6244a8509d4e7a7bba65c453802e5ca24dfcebd.
